src/api/controllers/moviesController.ts

```diff
@@ -1,4 +1,27 @@
-import { Router } from "express";
-const router : Router = Router()
+import { MovieRepository } from '../repo/movieRepository';
+import { MovieDTO } from 'MovieDTO';
 
-export const moviesController : Router = router
\ No newline at end of file
+export class MoviesController {
+    private repo: MovieRepository;
+    /**
+     * Init Repository for Movies
+     * @returns MovieRepository
+     */
+    public initDatabase() {
```

Comment:
Es lo que comentaba de la documentación. Si pones el código así se vuelve "auto-documentado"

```suggestion
    public initDatabase(): MovieRepository {
```